handlers/guildHandlers.js

```diff
@@ -2,33 +2,32 @@ function createMutedRole(guild) {
   if (guild.available) {
     guild.roles
```

Comment:
Now the bot seems to be creating multiple roles with the same name ('Muted') if the bot is removed/added to the server several times. It would be good to have a check here for a role with the name and not create a new role if it already exists.

Reply:
Good idea, implemented.